lib/lotus/utils/path_prefix.rb

```diff
@@ -82,7 +82,7 @@ def relative_join(strings, separator = @separator)
         prefix = @string.gsub(@separator, separator)
 
         self.class.new(
-          Utils::Kernel.Array([prefix, strings]).join(separator),
+          [prefix, strings].join(separator).chomp("/"),
```

Comment:
@AlfonsoUceda I can't recall. Have you guys tried to run all the other test suites against this branch? If all they pass, we can merge this.